Utils/update_playbook.py

```diff
@@ -13,12 +13,26 @@ def add_description(playbook):
     :return: updated playbook dict
     """
     for task_id, task in playbook.get("tasks", {}).items():
-        if task.get("type") in ["start", "end", "title"]:
+        if task.get("type") in ["start", "end", "title", "playbook"]:
             playbook["tasks"][task_id]["task"]["description"] = ""
 
     return playbook
 
 
+def update_playbook_task_name(playbook):
+    """
+    update the name of the task to be the same as playbookName it is running
+
+    :param playbook: playbook dict loaded from yaml
+    :return: updated playbook dict
+    """
+    for task_id, task in playbook.get("tasks", {}).items():
+        if task.get("type") == "playbook":
+            playbook["tasks"][task_id]["task"]["name"] = playbook["tasks"][task_id]["task"]["playbookName"]
```

Comment:
items returns the value by reference
```suggestion
            task["task"]["name"] = task["task"]["playbookName"]
```